keyboards/keychron/q4/ansi_v1/rules.mk

```diff
@@ -0,0 +1,29 @@
+# MCU name
+MCU = STM32L432
+
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+# Build Options
+#   change yes to no to disable.
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable USB N-key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+DIP_SWITCH_ENABLE = yes
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = CKLED2001
+RAW_ENABLE = yes
```

Comment:
This is only used in the via and keychron keymaps, correct? 

If so, then this should be removed from here (especially as `VIA_ENABLE` forces RAW_ENABLE.
```suggestion
```

Reply:
The `keychron` folder does utilize this flag for the custom keycodes, but not the `via` keymap since any reference of the custom code was removed as per tzarc's suggested changes in the PR for the Q3, and in the PR checklist which says how both, `default (and via) keymaps should be “pristine”`. Still, I'll disable this flag since you say that this acts like a double enable.

Reply:
Removed.